packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/SandboxMenu.tsx

```diff
@@ -178,7 +178,8 @@ export const SandboxMenu: React.FC<SandboxMenuProps> = ({
             actions.modals.moveSandboxModal.open({
               sandboxIds: [item.sandbox.id],
               preventSandboxLeaving:
-                item.sandbox.permissions.preventSandboxLeaving,
+                item.sandbox.permissions.preventSandboxLeaving ||
+                (!hasActiveSubscription && hasMaxPublicSandboxes),
```

Comment:
Should `hasMaxPublicSandboxes` also contain the logic of `hasActiveSubscription`? so it is always false for PRO accounts basically

Reply:
@alexnm yes, I think it's redundant to look at `hasActiveSubscription` every time we check for usage items.